apps/web/pages/[user].tsx

```diff
@@ -284,7 +287,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     avatar: `${WEBAPP_URL}/${user.username}/avatar.png`,
   }));
 
-  if (!users.length) {
+  if (!users.length || (!isValidOrgDomain && !users.some((user) => user.organizationId === null))) {
```

Comment:
Making sure selected users belong to the org inferred from subdomain if present.